XenAdmin/Wizards/NewSRWizard_Pages/Frontends/LVMoFCoE.cs

```diff
@@ -11,13 +11,20 @@ public LVMoFCoE()
 
         #region LVMoHBA overrides
 
-        public override SR.SRTypes SrType { get { return SR.SRTypes.lvmofcoe; } }
+        public override SR.SRTypes SrType { get { return srProvisioningMethod.Lvm ? SR.SRTypes.lvmofcoe : SR.SRTypes.gfs2; } }

```

Comment:
I think this is fine now, but probably !srProvisioningMethod.Gfs2 would be safer (when these controls would be invisible...)